fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchReverseChainTest.java

```diff
@@ -0,0 +1,1034 @@
+        assertResponse(response, Response.Status.CREATED.getStatusCode());
+
+        // Get the organization's logical id value.
+        organization1Id = getLocationLogicalId(response);
+
+        // Next, call the 'read' API to retrieve the new organization and verify it.
+        response = target.path("Organization/" + organization1Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+    }
+
+    @Test(groups = { "server-search-reverse-chain" }, dependsOnMethods = {"testCreateEndpoint"})
+    public void testCreateOrganization2() throws Exception {
+        WebTarget target = getWebTarget();
+
+       // Build a new Organization.
+        Organization organization = TestUtil.getMinimalResource(ResourceType.ORGANIZATION, Format.JSON);
+        organization = organization.toBuilder()
+                .name(of(tag))
+                .endpoint(Reference.builder().reference(of("Endpoint/" + endpointId)).build())
+                .build();
+
+        // Call the 'create' API.
+        Entity<Organization> entity = Entity.entity(organization, FHIRMediaType.APPLICATION_FHIR_JSON);
+        Response response = target.path("Organization").request().post(entity, Response.class);
+        assertResponse(response, Response.Status.CREATED.getStatusCode());
+
+        // Get the organization's logical id value.
+        organization2Id = getLocationLogicalId(response);
+
+        // Next, call the 'read' API to retrieve the new organization and verify it.
+        response = target.path("Organization/" + organization2Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+    }
+
+    @Test(groups = { "server-search-reverse-chain" }, dependsOnMethods = {"testCreateOrganization1"})
+    public void testCreatePatient1() throws Exception {
+        WebTarget target = getWebTarget();
+
+        // Build a new Patient and then call the 'create' API.
+        Patient patient = TestUtil.getMinimalResource(ResourceType.PATIENT, Format.JSON);
+        patient = patient.toBuilder()
+                .gender(AdministrativeGender.MALE)
+                .name(HumanName.builder()
+                    .given(of("1" + tag))
+                    .build())
+                .meta(Meta.builder()
+                    .tag(Coding.builder()
+                        .code(Code.of(tag))
+                        .build())
+                    .build())
+                .generalPractitioner(Reference.builder().reference(of("PractitionerRole/" + tag)).build())
+                .managingOrganization(Reference.builder().reference(of("Organization/" + organization1Id)).build())
+                .birthDate(Date.of(now.toString().substring(0,10)))
+                .telecom(ContactPoint.builder().system(ContactPointSystem.PHONE).value(of("1" + tag)).build())
+               .build();
```

Comment:
above is a good candidate for the formatter:off comment and end with formatter on

Reply:
not clear on this - just add a comment before and after with the annotation?

Reply:
That's right.  This was more of a heads up. 
It keeps it from getting reformatted accidently